### ISSUE TITLE
Avis Home Page: Purpose of the social media links in the bottom banner is not defined.

### ISSUE THREAD

**Issue by aishahussain24** [Kishan1601/Test-1#6]
Environment
windows 10
Chrome Version 98.0.4758.102
NVDA Version 2021.3.5

Repro Step
The purpose of the social media links are not defined and read by the screen reader.

Expected Result
The social media links should be defined and read by the screen reader.

Actual Result
social media links are not defined and read by the screen reader.

WCAG Guideline
https://www.w3.org/TR/WCAG21/#link-purpose-in-context
